<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Student_Dialog</class>
 <widget class="QDialog" name="Student_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>707</width>
    <height>639</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #73C2FB; /* Ice Blue*/
    font-family: 'Segoe UI', sans-serif;
}</string>
  </property>
  <widget class="QLineEdit" name="Welcome">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>70</y>
     <width>391</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI,sans-serif</family>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
 {
    color: #FAFAD2;
    font-size: 24px;
    font-weight: bold;
}</string>
   </property>
   <property name="text">
    <string>Welcome to Spartan Ride</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>0</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../SpartanLogo.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>10</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    background-color: #FAFAD2; /* Gold */
    color: #0B1D3A;
    border: none;
    border-radius: 10px;
    padding: 10px;
    font-weight: bold;
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../../Images/person.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="student_name">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>10</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
 {
    color: #FAFAD2;
    font-size: 12px;
    font-weight: bold;
}</string>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="Balamce">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>30</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
 {
    color: #FAFAD2;
    font-size: 12px;
    font-weight: bold;
}</string>
   </property>
   <property name="text">
    <string>Balance</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>125</y>
     <width>701</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="Student_pushbuttons">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>150</y>
     <width>701</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="Wallet">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #FAFAD2; /* Gold */
    color: #0B1D3A;
    border: none;
    border-radius: 10px;
    padding: 10px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #FAFAD2;
}</string>
    </property>
    <property name="text">
     <string>Wallet</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RideHistory">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #FAFAD2; /* Gold */
    color: #0B1D3A;
    border: none;
    border-radius: 10px;
    padding: 10px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #FAFAD2;
}</string>
    </property>
    <property name="text">
     <string>Ride History</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BookaRide">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #FAFAD2; /* Gold */
    color: #0B1D3A;
    border: none;
    border-radius: 10px;
    padding: 10px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #FAFAD2;
}</string>
    </property>
    <property name="text">
     <string>Book Ride</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Profile">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #FAFAD2; /* Gold */
    color: #0B1D3A;
    border: none;
    border-radius: 10px;
    padding: 10px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #FAFAD2;
}</string>
    </property>
    <property name="text">
     <string>Profile</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="Feedback">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>590</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI,sans-serif</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #FAFAD2; /* Gold */
    color: #0B1D3A;
    border: none;
    border-radius: 10px;
    padding: 10px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #FAFAD2;
}</string>
   </property>
   <property name="text">
    <string>Feedback</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SignOut">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>590</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI,sans-serif</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #FAFAD2; /* Gold */
    color: #0B1D3A;
    border: none;
    border-radius: 10px;
    padding: 10px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #FAFAD2;
}</string>
   </property>
   <property name="text">
    <string>Sign Out</string>
   </property>
  </widget>
  <widget class="QLabel" name="Schedule">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI,sans-serif</family>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
 {
    color: #FAFAD2;
    font-size: 24px;
    font-weight: bold;
}</string>
   </property>
   <property name="text">
    <string>My Schedule</string>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>391</width>
     <height>281</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QCalendarWidget {
    background-color: #E6F2FF; /* Light blue bg */
   
    border-radius: 10px;
    font-family: 'Segoe UI';
    font-size: 14px;
}

QCalendarWidget QToolButton {
    color: #0B1D3A; /* Dark navy */
    font-weight: bold;
    background-color: ;
    border: none;
    padding: 5px;
    border-radius: 5px;
}

QCalendarWidget QMenu {
    background-color: #FFFACD;
}

QCalendarWidget QSpinBox {
    width: 60px;
}
</string>
   </property>
   <property name="selectedDate">
    <date>
     <year>2025</year>
     <month>4</month>
     <day>9</day>
    </date>
   </property>
   <property name="selectionMode">
    <enum>QCalendarWidget::SingleSelection</enum>
   </property>
   <property name="navigationBarVisible">
    <bool>true</bool>
   </property>
   <property name="dateEditEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>230</y>
     <width>194</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>240</y>
     <width>221</width>
     <height>311</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../Images/ChatGPT Image Apr 21, 2025, 10_35_04 PM.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="HomeLabel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>590</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI,sans-serif</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #FAFAD2; /* Gold */
    color: #0B1D3A;
    border: none;
    border-radius: 10px;
    padding: 10px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #FAFAD2;
}</string>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
  </widget>
  <widget class="QLabel" name="namelabel">
   <property name="geometry">
    <rect>
     <x>519</x>
     <y>10</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="balancelabel">
   <property name="geometry">
    <rect>
     <x>519</x>
     <y>30</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
